packages/team-management/src/ducks/teams.ts

```diff
@@ -0,0 +1,63 @@
+/** Teams redux module */
+import {
+  fetchActionCreatorFactory,
+  removeActionCreatorFactory,
+  setTotalRecordsFactory,
+  reducerFactory,
+  getItemsByIdFactory,
+  getItemsArrayFactory,
+  getItemByIdFactory,
+  getTotalRecordsFactory,
+} from '@opensrp/reducer-factory';
+
+/** The reducer name */
+export const reducerName = 'teams';
```

Comment:
@KipSigei given that this is a model of the `Organization` openSRP api entitty, should we name it as so?

Reply:
Won't renaming this to `Organization` bring up contextual confusion?